Projekt 1 Philipp Mark/Program.cs

```diff
@@ -1,2 +1,95 @@
-﻿// See https://aka.ms/new-console-template for more information
-Console.WriteLine("Hello, SWP");
+﻿Console.WriteLine("Wählen sie eine Rechenoperation aus");
+Console.WriteLine("1: Wurzel");
+Console.WriteLine("2: Potenzieren");
+Console.WriteLine("3: Fakultät");
+Console.WriteLine("4: BMI");
+Console.WriteLine("5: Quadratische Flächen");
+Console.WriteLine("6: Volumen");
+
+
+int chooseOperator = int.Parse(Console.ReadLine()); 
+MathOperation operation = new MathOperation();
+
+switch (chooseOperator)
+{
+    case 1:
+
+        Console.WriteLine("Gebe ein von was du die Wurzel ziehen willst");
+        double wurzelInput = Convert.ToDouble(Console.ReadLine());
+        double wurzelErgebnis = Math.Sqrt(wurzelInput);
+        Console.WriteLine("Das Ergebnis ist: " + wurzelErgebnis);
+
+        break;
+
+    case 2:
+        Console.WriteLine("Gebe ein von was du Potenzieren willst!");
+        double basenInput = Convert.ToDouble(Console.ReadLine());
+        Console.WriteLine("Gebe deine Exponente ein");
+        double potenzInput = Convert.ToDouble(Console.ReadLine());
+        double potenzErgebnis = Math.Pow(basenInput, potenzInput);
+        Console.WriteLine("Das Ergebnis ist: " + potenzErgebnis);
+
+        break;
+    case 3:
+        Console.WriteLine("Schreibe die Zahl ein von der du die Fakultät herausfinden willst");
+        int fakultätInput = int.Parse(Console.ReadLine());
+        long fakultätErgebnis = operation.Fakultät(fakultätInput);
```

Comment:
Bitte keine ä's, ö's, ... in Variablennameb... auch nicht in Datei oder Verzeichnisnamen...